book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,472 @@
+
+
+(cl-inclusive-events-comms-jargon)=
+## Avoiding Jargon
+Research fields can often rely on acronyms and jargon to communicate effectively between colleagues. 
+However, the use of jargon can make the involvement and inclusion of people with lived experience difficult. 
+
+Reflect on how, and where, your research field uses jargon that might not be understandable to those outside your field. 
+Although you may be able to assume that many people with lived experience are familiar with any common acronyms or jargon used, it can benefit the inclusion of all attendees and experience levels if the amount of jargon used can be minimized. 
+For example, researchers, clinicians and people with a lived experience of arthritis may be more familiar with terms and medication names related to arthritis and associated conditions. 
+However, many attendees may be unfamiliar with these terms can can feel excluded from conversation and event content that focuses on them 
+
+As part of planning, consider how you can reduce the use of jargon and acronyms at your event, and create a strategy for supporting speakers and facilitators to also avoid unnecessary jargon. 
+
+You may want to develop a Glossary of Terms that can be included in information booklets, and where possible, work with researchers to be mindful of, and review, language choices to ensure inclusivity, such as avoiding gender-specific terms and including language that recognizes the diverse identities engaged in your event. 
+
+You can find online examples of Glossaries developed by research funding bodies, or review the [NHS list of jargon and acronyms](https://www.nhsconfed.org/publications/acronym-buster) for inspiration.
+
+(cl-inclusive-events-comms-plain-language)=
+### Using Plain Language
+Similar to avoiding jargon, ensure that your core communications are written using Plain Language or are in an "easy reading" format. 
+
+Using plain language is especially important in your registration and event information packs.
+
+Plain language, sometimes referred to as Plain English in the UK and North America, allows for communications and other information to be read and understood easier. 
+Consider how you will encourage attendees and speakers to adopt clear and plain language communication throughout your event. 
+
+For example, you could address and encourage the use of plain language during the opening address or by signposting it in your event's shared information. 
+You could also consider an activity where researchers are paired with non-researchers and are asked to explain their research in under 3 minutes. 
+This can help to bring "clear and effective communication" to the forefront of attendee's minds and set all attendees up to communicate clearly throughout the event.
+
+If possible, consider co-developing plain language and Easy Read content alongside a community representative or person with lived experience. 
```

Comment:
```suggestion
If possible, consider co-developing Plain Language and Easy Read content alongside a community representative or person with lived experience. 
```

What is the difference between Plain Language and Easy Read?

Reply:
me with noise cancelling headphones listening to Disney soundtracks and trying to stay out of discourse and drama 😅 😬 

Generally speaking, and if I don't have to link to any references, Easy Read was developed for people with intellectual disabilities or learning impairments. Compared to Plain Language which is more about reading comprehension level, sentence structure, clarity, and vocab use. 

I can dive into it, but it's something that I think would be better addressed in a dedicated section that has space to discuss the nuance of each. 

I was hoping to get away with not adding the discussion in here and that by both capitalizing it, and because the words "easy" and "read" themselves can be understood without much more explanation, it would mean that folks interested would then search for more information of "Easy Read" as a term, and that anyone reading at fae value would understand that "easy" "read" meant that the content would be easy to read.

Reply:
I think a short explanation, or indeed separate section, would be helpful! I did not come across Easy Read before I think so would love to learn more about it :)